src/base/memory_monitor_sr.cpp

```diff
@@ -251,5 +259,63 @@ namespace cubmem
     fflush (outfile_fp);
     fclose (outfile_fp);
   }
+
+#if (MMON_DEBUG_LEVEL == 1) || (MMON_DEBUG_LEVEL == 3)
+  void memory_monitor::print_debug_result ()
+  {
+    double mem_usage_ratio = 0.0;
+    FILE *outfile_fp = fopen ("mmon_debug_result.txt", "w+");
```

Comment:
Is this intended to be used temporarily? If not, wouldn't it be better to place this file in $CUBRID/log/server/?

Reply:
We need to handle the case where the outfile_fp value is not properly obtained due to an fopen error. For example, it could be set to stderr.

Reply:
done.